plugins/Installation/lang/en.json

```diff
@@ -150,6 +150,6 @@
         "WelcomeToCommunity": "Welcome to the Matomo community!",
         "CannotConnectToDb": "Cannot connect to the database",
         "CannotConnectToDbResolvingExplanation": "This may be a temporary issue, try %1$srefreshing the page%2$s. If the problem persists please contact your Matomo administrator.",
-        "EmailPrivacyNotice": "Your email address will be only used to send you the Matomo newsletter. You can unsubscribe at any time by clicking the unsubscribe link. We will not share your email with anyone else or use your email for any other purpose than sending you the Matomo newsletter. Please consult our %1$sprivacy policy%2$s for more information."
+        "EmailPrivacyNotice": "Your email address will be only used to send you the Matomo newsletter. You can unsubscribe at any time by clicking the unsubscribe link. We are using Mad Mimi for sending our newsletter. We will not share your email with anyone else or use your email for any other purpose than sending you the Matomo newsletter. Please consult our %1$sprivacy policy%2$s for more information."
```

Comment:
LGTM. Maybe we could mention the domain name without a link, so it's clear which service we use: `We are using Mad Mimi (madmimi.com) for sending our newsletter.` (although it's listed in the privacy policy that is linked, so may not be needed).

Reply:
@mattab just checking to confirm you also had a look at https://github.com/matomo-org/matomo/issues/15182#issuecomment-633854325

Reply:
Not sure what bit you refer to, but maybe  could clarify further to `We are using a third party service called Mad Mimi (madmimi.com) to store your email address and send our newsletter.` ?

Reply:
> We will not share your email with anyone else or use your email for any other purpose than sending you the Matomo newsletter

Sounds like we won't share the email with anyone else. But it will be shared with MadMimi. Should be mentioned more clearly maybe. Maybe also that this can change in the future. Could use eg 

`For this we have to share your email with a third party provider. Currently, we use Mad Mimi (a GoDaddy company), but this may change in the future. `

Reply:
Might also add the email will be shared basically with our hoster which is currently AlwaysData? In our privacy notices on the homepage we say it will be shared with our hosting provider

Reply:
`For this we have to share your email with a third party provider. Currently, we use Mad Mimi (a GoDaddy company), but this may change in the future. ` sounds good :+1: